Build.ps1

```diff
@@ -299,6 +299,13 @@ else
     exit
 }
 
+if ($IncludeExperimental -eq $true) {
+    # restore any missing PS packages
+    .\Download-PS-Prerequisites.ps1 `
+        -SitecoreUsername $SitecoreUsername `
```

Comment:
You also need to pass in `InstallSourcePath`  - in case someone specified a custom path.

There's also a logic flaw in Download-PS-Preqrequisites.ps1 (fixed [here](https://github.com/Sitecore/docker-images/blob/b26c91cf3ed20d705aed892b1700a51d4e6ca75b/build/Download-Module-Prerequisites.ps1#L90))

Reply:
Good catch. I took this code block in the INSTRUCTIONS.md file example.